spec/controllers/organizations_controller_spec.rb

```diff
@@ -5,6 +5,7 @@
 
   let(:organization)  { GitHubFactory.create_owner_classroom_org }
   let(:user)          { organization.users.first                 }
+  let(:student)       { GitHubFactory.create_classroom_student   }
```

Comment:
Ah that explains it, so the student is not supposed to be a member of the organization. It is supposed to be just another GitHub user